### DISCUSSION TITLE
Request for 2 monitored values in task bar and in game overlay

### DISCUSSION THREAD

**Discussion by AlejandroFlorin** [Rem0o/FanControl.Releases#1209] [Ideas]
Thanks for the best fan control UI out there. Much appreciated.

I'd like to make a couple of requests/suggestions though - to allow two monitored values in the task bar instead of just one.

Also, an option to display the two values as an always on top in-game overlay would be great.

Thanks again for such a great tool.

**Comment by GuiLL31** [comment]
sounds great

**Comment by cloudpatata-design** [comment]
@Rem0o this can be done over RivaTunerStatisticsServer (RTSS).
